apm-agent-plugins/apm-kafka-plugin/apm-kafka-headers-plugin/src/test/java/co/elastic/apm/agent/kafka/KafkaIT.java

```diff
@@ -271,8 +290,8 @@ public void testIgnoreTopic() {
 
         // we expect only one span for polling the reply topic
         List<Span> spans = reporter.getSpans();
-        assertThat(spans).hasSize(1);
-        verifyPollSpanContents(spans.get(0));
+        assertThat(spans.size()).isGreaterThanOrEqualTo(1);
+        verifyPollSpanContents(spans);
```

Comment:
[question] I'm not sure to understand here why we now have 1+ instead of exactly one previously.